docs/team.rst

```diff
@@ -2,134 +2,114 @@ Read the Docs Team
-Every day at least one of them has an issue that needs to be addressed by a site admin.
-This might include tasks like:
-
-* Resetting a password
-* Asking for a project name to be released
-* Troubleshooting build errors
-
-Support team members
-~~~~~~~~~~~~~~~~~~~~
-
-* `Eric Holscher`_ (Pacific Time)
-* `Anthony Johnson`_ (Mountain Time)
-* `Manuel Kaufmann`_ (Central Time)
-* Your Name Here
-
-**Please don't email us personally for support on Read the Docs.** You can email support@readthedocs.org for any issues you may have. 
+Today we:
 
-Joining the support team
-~~~~~~~~~~~~~~~~~~~~~~~~
+* Serve over **55 million pages** of documentation a month
+* Serve over **40 TB** of documentation a month
+* Host over **80,000 open source projects** and support over **100,000 users**
 
-The best place to start would be to start addressing some of the issues in our issue tracker.
-We have our support policies quite well documented in our :doc:`/contribute`.
-**Be bold**.
-Start trying to reproduce issues that people have,
-or talk to them to get more information.
-After you get the hang of things,
-we'll happily give you the ability to tag and close issues by joining our Support Team.
-
-
-Operations Team
----------------
-
-readthedocs.org is a service that millions of people depend on each month.
-As part of operating the site,
-we maintain a 24/7 on-call rotation.
-This means that folks have to be available and have their phone in service.
-
-Ops team members
-~~~~~~~~~~~~~~~~
-
-* `Eric Holscher`_ (Pacific Time)
-* `Anthony Johnson`_ (Mountain Time)
-* `Matt Robenolt`_ (Pacific Time)
-* Your Name Here
-
-Feel free to ask any of us if you have questions or want to join!
-
-Joining the ops team
-~~~~~~~~~~~~~~~~~~~~
-
-We are always looking for more people to share the on-call responsibility.
-If you are on-call for your job already,
-we'd love to piggy back on that duty as well.
+Read the Docs is provided as a free service to the open source community,
+and we hope to maintain a reliable and stable hosting platform for years to come.
 
-You can email us at dev@readthedocs.org if you want to join our operations team.
-Because of the sensitive nature (API tokens, secret keys, SSL certs, etc.) of the work,
-we keep a private GitHub repository with the operations code & documentation.
+We mainly fund our operations through :doc:`advertising </advertising/index>`
+and corporate-hosted documentation with `Read the Docs for Business <https://readthedocs.com/>`_,
+and we are supported by a number of generous :doc:`sponsors </sponsors>`.
 
-The tools that we use are:
+The :ref:`Core Team <core-team>` works full time on the service,
+and we are also honored to have several :ref:`External Contributors <external-contributors>`.
 
-* Salt
-* Nagios
-* Graphite/Grafana
-* Nginx
-* Postgres
-* Django
-* Celery
+.. _core-team:
 
-It's fine if you aren't familiar with all of these things,
-but are willing to help with part of it!
+Core Team
```

Comment:
This ordering and heading feels weird. I'd probably do something like:

```
# Paid staff
Paragraph explaining that we're paid to work on it
Images & team descriptions
# Teams
Listing of teams
# External contributors
```

Reply:
Done!